### PR TITLE
New example for using emotion and PrismJS plugin

### PR DESCRIPTION
**Pull request by arwyatt**
This example uses emotion and the gatsby-remark-prismjs plugin to style code snippet highlighting.  Since the existing demo for emotion didn't use markdown this seemed like it was a good candidate for its own example.

### CODE REVIEW

File: examples/using-emotion-prismjs/src/utils/typography.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Typography from "typography"
+import bootstrapTheme from "typography-theme-bootstrap"
+console.log(bootstrapTheme)
+
+bootstrapTheme.googleFonts = [{
+    name: `Source Sans Pro`,
```
Comment (KyleAMathews) [inline examples/using-emotion-prismjs/src/utils/typography.js]: Did you mean `Source Code Pro`?

**Comment by arwyatt** [comment]
I pushed the fix for the Source Code Pro typo. Thanks for catching that!

File: examples/using-emotion-prismjs/src/utils/typography.js (hunk 1 above)
Comment (arwyatt) [reply examples/using-emotion-prismjs/src/utils/typography.js]: I did. This is fixed now. Thanks for catching that!

**Comment by arwyatt** [comment]
Source Code Pro typo fixed!

**Comment by KyleAMathews** [comment]
Thanks! Sorry for the slow merge. Been conferencing lately.

**Comment by jlengstorf** [comment]
Hiya @arwyatt! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**